cli/util.go

```diff
@@ -37,8 +37,22 @@ func ParseAddress(s string) (loom.Address, error) {
 	return loom.Address{ChainID: TxFlags.ChainID, Local: loom.LocalAddress(b)}, nil
 }
 
+func ResolveAddressv1(s string, callFlags *ContractCallFlags) (loom.Address, error) {
```

Comment:
Instead of adding another version modify the original `ResolveAddress` to:
```golang
func ResolveAddress(s string, chainID, uri string)
```
And update the existing call sites.